pkgs/development/python-modules/etils/skip-failing-tests.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/etils/edc/dataclass_utils_test.py b/etils/edc/dataclass_utils_test.py
```

Comment:
Why are we not using disabledTests?

Reply:
Only because I wasn't aware of it, thanks for pointing it out. Updated to remove the patch in favor of `disabledTests`.